packages/docs/stories/components/checkbox.stories.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable */
+
+/* eslint-disable import/no-relative-packages */
+
+import '../../../components/src/components/checkbox/checkbox';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+import docsTokens from '../../../tokens/src/figma-tokens/_docs.json';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../src/helpers/component.js';
+const { args, argTypes } = storybookDefaults('syn-checkbox');
+const { overrideArgs } = storybookHelpers('syn-checkbox');
+const { generateTemplate } = storybookTemplate('syn-checkbox');
+
+const meta: Meta = {
+  component: 'checkbox',
+  args,
+  argTypes,
+  title: 'Components/syn-checkbox',
+  parameters: {
+    docs: {
+      description: {
+        component: docsTokens?.components?.['checkbox']?.default?.description?.value ?? 'No Description',
+      },
+    }
+  }
+};
+export default meta;
+
+type Story = StoryObj;
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: docsTokens?.components?.['checkbox']?.default?.description?.value ?? 'No Description',
+      }
+    }
+  }
+} as Story;
+
+
+/**
+ * Use the checked attribute to activate the checkbox.
+ */
```

Comment:
Please replace this with the stuff coming from Figma: `docsTokens?.components?.['checkbox']?.checked?.description?.value ?? 'No Description',`

etc.

Same for all other stories where the content is declared.

--

In other components (I believe e. g. in the button) we created a single function at the beginning of the story, to make this a  bit more fluent to write.

Reply:
There is currently only the default, it it is not namespaced like when using button, see https://github.com/SickDesignSystem/synergy/blob/feat/72-syn-checkbox/packages/tokens/src/figma-tokens/_docs.json#L215. I think @LeaLeRose would have to add more doc tokens to make this work.

Reply:
Done in latest commit.